src/test/java/Pages/LoginPage.java

```diff
@@ -0,0 +1,70 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.CacheLookup;
+import org.openqa.selenium.support.FindBy;
+
+public class LoginPage extends BasePage {
+
+    public LoginPage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    /*  @FindBy annotation is used at the attribute level
+         to identify web elements in a page
+        @CacheLookup is used to cache the web element location results
+         (since the web page is static and its content will not change)
+     */
+    // web elements
+    @FindBy(css = "[type='email']")
+    @CacheLookup
+    WebElement emailField;
+
+    @FindBy(css = "[type='password']")
+    @CacheLookup
+    WebElement passwordField;
+
+    @FindBy(css = "[type='submit']")
+    @CacheLookup
+    WebElement submitBtn;
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```